backend/model/top_container.rb

```diff
@@ -79,7 +79,8 @@ def collections
       else
         obj
       end
-    }.uniq {|obj| obj.uri}
+    }.compact.uniq {|obj| obj.uri}
```

Comment:
decided this one is ok - if obj doesn't have a root_record_id (not part of a resource) then it is nil, and that's ok because it isn't part of a collection. so compacting is alright.